run/test/specs/message_requests_clear_all.spec.ts

```diff
@@ -22,13 +24,12 @@ async function clearAllRequests(platform: SupportedPlatformsType) {
   // Select Clear All button
   await device2.clickOnByAccessibilityID('Clear all');
   await sleepFor(1000);
-  await runOnlyOnAndroid(platform, () =>
-    device2.clickOnElementAll({ strategy: 'accessibility id', selector: 'Yes' })
-  );
-  await runOnlyOnIOS(platform, () => device2.clickOnByAccessibilityID('Clear'));
+  await device2.clickOnByAccessibilityID('Clear');
+  // "messageRequestsNonePending": "No pending message requests",
+  const messageRequestsNonePending = localize('messageRequestsNonePending').strip().toString();
   await device2.waitForTextElementToBePresent({
     strategy: 'accessibility id',
-    selector: 'No pending message requests',
+    selector: messageRequestsNonePending as AccessibilityId,
```

Comment:
I understand why all of those as are needed. This is going to cause issues down the line...

Reply:
What issues? Maybe there's a better way to do it then..

Reply:
Have updated this behaviour in a different PR

Reply:
Have linked above aswell, https://github.com/burtonemily/session-appium/commit/a1aca60ba2395be4796f26612b5994155e7149e9